docs/dev/store.md

```diff
@@ -0,0 +1,163 @@
+# Store
```

Comment:
"Store" needs explanation

Reply:
Amended that to more explicitly refer to `pkg/store`.